aptos-move/e2e-move-tests/tests/rotate_auth_key.rs

```diff
@@ -9,26 +9,10 @@ use aptos_types::{
     transaction::authenticator::AuthenticationKey,
 };
 
+use aptos::account::key_rotation::RotationProofChallenge;
```

Comment:
Kinda weird that move e2e tests are dependent on the CLI, but I guess this is how it was set up for packaging